Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/operations/TaskOperation.ts

```diff
@@ -0,0 +1,25 @@
+import {IVariation,IExperiment} from './../models/IOptimize';
```

Comment:
Remove IVariation import if not using in file. Check and remove unused imports in all file please.